doc/release/1.14.1-notes.rst

```diff
@@ -2,38 +2,87 @@
 NumPy 1.14.1 Release Notes
 ==========================
 
-This is a bugfix release for some problems found since 1.14.0. The most
-important fixes are the reversion of the multifield index "view" behavior (see
-compatibility notes), and fixes to the spacing in the str and repr of complex
-values.
+This is a bugfix release for some problems reported following the 1.14.0 release. The major
+problems fixed are the following.
 
-The Python versions supported are 2.7 and 3.4 - 3.6. The Python 3.6 wheels
-available from PIP are built with Python 3.6.2 and should be compatible with
-all previous versions of Python 3.6. It was cythonized with Cython 0.26.1,
-which should be free of the bugs found in 0.27 while also being compatible with
-Python 3.7-dev.
+* Problems with the new array printing, particularly the printing of
+  complex values,
+* Problems with ``np.einsum`` with the new ``optimized=True`` default. Some
+  fixes for optimization have been applied and ``optimize=False`` is now the
+  default.
```

Comment:
Perhaps "is once more the default", or something emphasizing we're back to the 1.13 behavior?

Reply:
This was also in 1.13, indeed, since the `axis` parameter was introduced.

Reply:
What I mean is I think we should be explicit here that 1.14.1 restores the 1.13.3 behavior